yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/yb-voyager-pg-gather-assessment-metadata.sh

```diff
@@ -172,6 +186,34 @@ main() {
         fi
     fi
 
+    null_analyze_schemas=$(psql $pg_connection_string -tAqc "SELECT DISTINCT schemaname 
+FROM pg_stat_all_tables pgst1 
+WHERE schemaname = ANY(ARRAY['public','test_schema1'])
+  AND NOT EXISTS (
```

Comment:
hmm, I'm wondering - do we need a subquery. Can we not just do SELECT DISTINCT schemaname .... where schemaname IN () AND last_analyze IS NOT NULL?